packs/data/actions.db/mark-for-death.json

```diff
@@ -16,7 +16,141 @@
+                "option": "marked-for-death",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.SpecificRule.MarkForDeath.SeekMark",
+                "predicate": {
+                    "all": [
+                        "action:seek",
+                        "marked-for-death"
+                    ]
+                },
+                "selector": "perception",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.SpecificRule.MarkForDeath.DeceiveMark",
+                "predicate": {
+                    "all": [
+                        "action:feint",
+                        "marked-for-death"
+                    ]
+                },
+                "selector": "deception",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "definition": {
+                    "any": [
+                        "weapon:trait:agile",
+                        "weapon:trait:finesse"
+                    ],
+                    "not": [
+                        "weapon:trait:deadly-d6",
+                        "weapon:trait:deadly-d8",
+                        "weapon:trait:deadly-d10",
+                        "weapon:trait:deadly-d12"
+                    ]
+                },
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": {
+                    "all": [
+                        "marked-for-death"
+                    ]
+                },
+                "property": "weapon-traits",
+                "value": "deadly-d6"
+            },
+            {
+                "definition": {
+                    "any": [
+                        "weapon:trait:finesse",
+                        "weapon:trait:agile"
+                    ]
+                },
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": {
+                    "all": [
+                        "marked-for-death"
+                    ]
+                },
+                "property": "traits",
+                "value": "backstabber"
+            },
+            {
+                "definition": {
+                    "all": [
+                        "weapon:trait:deadly-d6"
+                    ],
+                    "any": [
+                        "weapon:trait:agile",
+                        "weapon:trait:finesse"
+                    ]
+                },
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": {
+                    "all": [
+                        "marked-for-death"
+                    ]
+                },
+                "property": "weapon-traits",
+                "value": "deadly-d8"
+            },
```

Comment:
Can you verify in Foundry that this will not result in the weapon having _both_ deadly d6 and deadly d8, and thus rolling too many dice?

Reply:
Can confirm.
First roll without Toggle, 2nd with.
![image](https://user-images.githubusercontent.com/95336883/174524807-dc24c3e5-eab0-4cf2-b5a1-8975a018d029.png)

Reply:
It looks like the portions that upgrade the deadly die just cascade, and everything loads up to Deadly D12.

Would brackets perhaps work, or should we remove the automation for the upgrade, and leave notes on how to upgrade it in the RE?

Reply:
Alternatively, I can leave in the Deadly D6 > Deadly D8, and we can add notes to change that RE if Deadly D8 or higher is already on, thus only having 2 `AdjustStrike` REs.

Reply:
Reverse the order of the Adjust Strikes. Have the one for advancing to d12 first, then go down from there...
Check the Discord, I'll @ you with a modified action file to import and test.